dist/package.json

```diff
@@ -30,10 +30,10 @@
   "license": "Apache-2.0",
   "peerDependencies": {
     "react": ">=15 <=17",
-    "firebase": "^8.2.4"
+    "firebase": "^9.0.0"
```

Comment:
Let's go with the same peer as firebaseui for safety, `^9.1.3`, as there were some bugs that needed addressing.

Reply:
Fix:
https://github.com/firebase/firebaseui-web-react/blob/e9cf37bf5a8b621d92399ed4ca39185ecec65a50/package.json#L33